package.json

```diff
@@ -68,7 +68,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-unused-imports": "^3.0.0",
     "ethers": "^5.8.0",
-    "jest": "^29.6.4",
+    "jest": "^29.7.0",
```

Comment:
Updated jest, it helped a bit with the error in bignumer, but didn't solve it in all cases